auto_cpufreq/utils/config_event_handler.py

```diff
@@ -0,0 +1,20 @@
+import pyinotify
+
+class ConfigEventHandler(pyinotify.ProcessEvent):
+    def __init__(self, config) -> None:
+        self.config = config
+
+    # activates when file is modified
```

Comment:
I would be explicit and replace "file" with "auto-cpufreq config file" so when someone is reading the code without any context what this is about.